README.md

```diff
@@ -266,7 +266,7 @@ By default, profiles are created with autoconnect enabled.
 The `mac` address is optional and restricts the profile to be usable only on
 devices with the given MAC address. `mac` is only allowed for `type`
 `ethernet` or `infiniband` to match a non-virtual device with the
-profile.
+profile. The value of `mac` address is string and should use with quotation mark.
```

Comment:
Please put the documentation update in its own PR, then we can merge it by itself.

Reply:
Seperate PR for docs,
https://github.com/linux-system-roles/network/pull/367